src/snowflake/cli/api/project/schemas/entities/streamlit_entity_model.py

```diff
@@ -65,9 +65,20 @@ def artifacts_must_exists(self):
             return self
 
         for artifact in self.artifacts:
-            if not artifact.exists():
+
+            if artifact.is_file() and not artifact.exists():
                 raise ValueError(
                     f"Specified artifact {artifact} does not exist locally."
                 )
-
         return self
+
+    def get_artifacts(self):
+        _artifacts = []
+        for artifact in self.artifacts:
+            if "*" in str(artifact):
+                root = artifact.parent.absolute() if "**" not in str(artifact) else Path(".")
```

Comment:
So, is this also supposed to support `**/*`-style paths? That would be excellent!

Reply:
Yes, it should